packages/RadioCard/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@tds/community-radio-card",
+  "version": "0.1.0-0",
+  "description": "",
+  "main": "index.cjs.js",
+  "module": "index.es.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/telus/tds-community.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "TELUS digital",
+  "engines": {
+    "node": ">=8"
+  },
+  "bugs": {
+    "url": "https://github.com/telus/tds-community/issues"
+  },
+  "homepage": "http://tds.telus.com",
+  "peerDependencies": {
+    "react": "^16.8.2",
+    "react-dom": "^16.8.2",
+    "styled-components": "^4.1.3"
+  },
+  "dependencies": {
+    "@tds/core-box": "^2.1.3",
+    "@tds/core-colours": "^2.2.1",
+    "@tds/core-heading": "^3.0.4",
+    "@tds/util-helpers": "^1.5.0",
+    "@tds/shared-styles": "^1.5.2",
+    "prop-types": "^15.6.2"
```

Comment:
Can you add dev dependancy for core-paragraph

Reply:
Done :)